source/_UIAHandler.py

```diff
@@ -194,7 +198,7 @@ def MTAThreadFunc(self):
 						pass
 				# Windows 10 RS5 provides new performance features for UI Automation including event coalescing and connection recovery. 
 				# Enable all of these where available.
-				if isinstance(self.clientObject,IUIAutomation6):
+				if False: #isinstance(self.clientObject,IUIAutomation6):

```

Comment:
Do you intend this to stay? I guess a decision about coalescing has to be made here:

1. Either use event coalescing with global property change event registration or coalescing with per foreground events. I'd like to opt in for the latter
2. I assume this change can still considered somewhat risky. For easier debugging, could it help to add some additional config parameters to the UIA section? Like: "useEventCoalescingWhenAvailable" (defaults to True) and useGlobalUIAPropertyChangeEvents (defaults to False). This way, we can easily change defaults if one or both of the features causes problems and we don't have to necessarily revert the whole pr.